workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/load_diagnostic_data.py

```diff
@@ -87,15 +87,30 @@ def _load_prognostic_run_3d_output(url: str):
     prognostic_3d_output = [
         item
         for item in fs.ls(url)
-        if item.endswith("3d.zarr") or item.endswith("state_after_timestep.zarr")
+        if item.endswith("diags_3d.zarr") or item.endswith("state_after_timestep.zarr")
     ]
     if len(prognostic_3d_output) > 0:
         outputs = []
         for item in prognostic_3d_output:
             zarr_name = os.path.basename(item)
             path = os.path.join(url, zarr_name)
             outputs.append(_load_standardized(path))
-        return xr.merge(outputs)
+        merged = xr.merge(outputs)
+        # interpolation is slow, so only keep a subset of 3D outputs
+        keep_vars = [
```

Comment:
could you change this subsetting to use the standard `subset_variables` transform that is used for all the other diagnostics? i.e. do it as part of the diagnostic calculation instead of the loading. Shouldn't make too much of a efficiency difference since data is loaded lazily (but let me know if the speed is hugely different... maybe even setting up the task graph for the interpolation is really slow?)

Reply:
e.g. like this: https://github.com/VulcanClimateModeling/fv3net/blob/149b545c5dd633cae746f938e84fb82906623bc0/workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/compute.py#L244